rust/operator-binary/src/format/well_known.rs

```diff
@@ -123,6 +123,18 @@ pub struct CompatibilityOptions {
     pub tls_pkcs12_password: Option<String>,
 }
 
+/// Options to customize the well-known format file names.
+///
+/// The fields will either contain the default value or the custom user-provided one. This is also
+/// the reason why the fields are not wrapped in [`Option`].
```

Comment:
I'd rather provide this defaulting on `NamingOptions` itself than refer to some nebulous  defaulting elsewhere.